README.md

```diff
@@ -85,4 +85,4 @@ double quotes).
 ### Merge policy
 
 If you can commit to this repository, see the
-[WHATWG team](https://github.com/whatwg/meta/blob/master/TEAM.md) guidelines.
+[WHATWG team](https://github.com/whatwg/meta/blob/master/MAINTAINERS.md) guidelines.
```

Comment:
Should we make this `WHATWG maintainers` ?

Reply:
Sounds good to me. @domenic?

Reply:
Sounds good

Reply:
Fixed in https://github.com/whatwg/infra/pull/186/commits/93cc2ed895522f0dad1731b9ad311ebca7ba6bb4